### ISSUE TITLE
Stops Smelting

### ISSUE THREAD

**Issue by KohlrabitoDev** [A5ho9999/MinecraftMods#18]
### Mod name

Smelter The Hedgehog

### Mod Version

v1.1.0

### Minecraft Version

1.21.1

### Fabric Versions

Fabric Loader: 0.16.2 | API: fabric-api-0.102.1+1.21.1

### Confirmation Checkboxes

- [X] I have confirmed that this issue can be reproduced with minimal setup (Not a large Modpack)
- [X] I am NOT using a 3rd Party Client (Feather Client, Luna Client, etc)

### Issue Description

Furnace just stops smelting after 16 blocks or so, typically just after a piece of coal is done smelting. Can be resumed by just removing whatever you're trying to smelt, then putting it back. This only happens after editing the config as I did here:
```
{
	"ModEnabled": true,
	"Furnace": {
		"CookTime": 20,
		"FuelEfficiency": 200
	},
	"BlastFurnace": {
		"CookTime": 15,
		"FuelEfficiency": 100
	},
	"Smoker": {
		"CookTime": 15,
		"FuelEfficiency": 100
	},
	"Other": {
		"CookTime": 15,
		"FuelEfficiency": 200
	}
}
```
[https://gyazo.com/9d887e2724a81a47174d8d46a5e17f71](https://gyazo.com/9d887e2724a81a47174d8d46a5e17f71)

### Steps to reproduce

1. Edit config.
2. Smelt a stack of something normally.
3. It just stops smelting as if it ran out of fuel.
4. Refresh it by moving either the fuel or what you're trying to smelt.

### Expected Behavior

It should just keep smelting right through the entire stack.

### Crash Report / Log

No crash report

**Comment by A5ho9999** [comment]
I am unable to reproduce this, could you please provide a full mod list as well

**Comment by KohlrabitoDev** [comment]
Yes, although I didn’t have any time to cull through the junk so a lot of the stuff with the double hyphens are just junk. I also should’ve mentioned this is all on a server.
https://mclo.gs/giKNegz

**Comment by A5ho9999** [comment]
Okay this would be an issue with lithium, I'll release a fix in a couple days. In the meantime you can edit the lithium properties file to add `mixin.world.block_entity_ticking=false` at the bottom of the file as a temporary measure

**Comment by KohlrabitoDev** [comment]
Yes that worked, thank you!

**Comment by A5ho9999** [comment]
Fix for this has been released in [v1.1.1](https://modrinth.com/mod/smelter-the-hedgehog/version/1.1.1-1.21.0) as well as small bug fix for another issue found from testing.
